ote/src/cljs/ote/app/controller/transit_visualization.cljs

```diff
@@ -104,7 +104,8 @@
                       (dissoc :changes-route-no-change)
                       (assoc :service-changes-for-date-loading? true)
                       (assoc :all-route-changes-display? false)
-                      (assoc-in [:open-sections :gtfs-package-info] false)))]
+                      (assoc-in [:open-sections :gtfs-package-info] false)
+                      (dissoc :selected-route)))]
```

Comment:
I think that this will fix the issue, but I was wondering if we could clean up that :transit-visualization key completely?
Something like this

(fn [transit-visualization]
                  (-> transit-visualization	                  (-> transit-visualization
                      (assoc :all-route-changes-checkbox nil)	                      (assoc :all-route-changes-checkbox nil)
                      (assoc :all-route-changes-display? nil)	                      (assoc :all-route-changes-display? nil)                                        
                      (assoc :service-changes-for-date-loading? true)	                      (assoc :service-changes-for-date-loading? true)
                      (assoc :all-route-changes-display? false)	                      (assoc :all-route-changes-display? false)
                      (assoc-in [:open-sections :gtfs-package-info] false)))]	                      (assoc-in [:open-sections :gtfs-package-info] false)))

(assoc app :transit-visualization init-tv)

Reply:
Can't make this out, was there a copy-paste problem here? @rckopteri 

I agree that cleaning all keys should be done whenever entering a view. Whether to `dissoc` or `nil` them is a matter of how logic is implemented, but generally speaking dissocing sounds better if there's no reason to do otherwise.

Reply:
Let's try if this helps. @jkorpelin noticed that on a slower machine / browser there are still some glitches when navigating to the view a second time.